lib/mspec/commands/mspec.rb

```diff
@@ -93,6 +93,20 @@ def multi_exec(argv)
     require 'mspec/runner/formatters/multi'
     formatter = MultiFormatter.new
 
+    cores = cores()
+    if /(?:\A|\s)--jobserver-(?:auth|fds)=(\d+),(\d+)/ =~ ENV["MAKEFLAGS"]
+      begin
+        r = IO.for_fd($1.to_i(10), "rb", autoclose: false)
+        w = IO.for_fd($2.to_i(10), "wb", autoclose: false)
+      rescue
+        r.close if r
+        nil
+      else
+        jobserver = [r, w]
+        jobtokens = r.read_nonblock(1024)
+        cores = jobtokens.size
+      end
+    end
```

Comment:
Is there an advantage from getting it from `make` flags over getting it via e.g. `/proc/cpuinfo`?